mne/source_space.py

```diff
@@ -1132,7 +1132,52 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
     # take point locations in MRI space and convert to MNI coordinates
     xfm = _read_talxfm(subject, subjects_dir, mode)
     data = np.array([rr[h][v, :] for h, v in zip(hemis, vertices)])
-    return apply_trans(xfm['trans'], data)
+    return apply_trans(xfm['trans'], data), xfm
```

Comment:
This is an API change. If you need the `xfm` internally you can re-read it using `_read_talxfm` (it should be fast anyway)